packages/harmony/src/components/notification-count/NotificationCount.tsx

```diff
@@ -0,0 +1,114 @@
+import { useTheme, CSSObject } from '@emotion/react'
+import numeral from 'numeral'
+
+import { Flex } from '../layout/Flex'
+import { Text } from '../text/Text'
+
+import type { NotificationCountProps } from './types'
+
+/**
+ * Formats a count into a more readable string representation.
+ * For counts over 1000, it converts the number into a format with a suffix (K for thousands, M for millions, etc.)
+ * For example:
+ * - 375 => "375"
+ * - 4,210 => "4.21K"
+ * - 443,123 => "443K"
+ * - 4,001,000 => "4M"
+ * If the count is 0, it returns "0".
+ * This function is pulled over from the common package because we don't use the common package in Harmony.
+ */
+export const formatCount = (count: number) => {
```

Comment:
ah interesting since format count is in common... check if there is a formatCount elsewhere in harmony, otherwise lets add it in the utils section rather inline here

Reply:
if you have any good ideas how to rectify formatCount existing in common + harmony + web, lmk